models.py

```diff
@@ -1158,6 +1190,50 @@ def determine_severity(self):
                 raise ValueError(f"Outcome code '{self.name}' not recognized")
 
 
+class Version(TimestampedBaseModel):
+
+    """
+    A model to store and track Validation Service software versions.
+    """
+
+    id = models.AutoField(
+        primary_key=True,
+        help_text="Identifier of the Version (auto-generated)."
+    )
+
+    name = models.CharField(
```

Comment:
Did you consider multiple fields, e.g. major, minor, patch, fix, to capture the version information?  Perhaps using a string field provides flexibility for potentially migrating away from rigid SemVer.

Reply:
We could have textual versions like `pre-0.6` or `0.6.9-alpha` or something. This was the easiest just to capture version/timestamp combos. We can always go more granular if needed in `IVS-203`